apps/client/src/app/app.tsx

```diff
@@ -32,33 +35,38 @@ export function App() {
   const clientRecaptchaChecker: IRecaptchaChecker = isLocalEnvironment ?
     new LocalRecaptchaChecker() :
     new ClientRecaptchaChecker(axiosInstance);
-  const firebaseAuthenticationService = isLocalEnvironment ?
+  const authenticationAuthenticationService = isLocalEnvironment ?
```

Comment:
The naming is wrong. `authenticationAuthenticationService` ? 😂

Reply:
Hahhaha, it was firebaseAuthenticationService, and I wanted to make it abstract from firebase, I renamed it without looking 😂, that's what we call abstraction overwhelming